docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -783,55 +781,54 @@ h|Kafka property name
 * https://kafka.apache.org/documentation/#topicconfigs[Topic-Level Configs^] in Kafka documentation
 
 [id="con-using-kafka-topics-with-registry-schemas_{context}"]
-== Using topics in {product-long-kafka} with schemas in {product-long-registry}
+== Schemas in {product-registry} for topics in {product-kafka}
```

Comment:
Yeah, per my previous comment, this section was added by me. The concept module is effectively acting as an assembly within the main assembly, hence the gerund title. I might even have discussed this approach with you prior to doing it. It was preferable to creating a standalone guide, because the content fit quite well within this one.

So, I think we'd need to restore the original title if we keep this content where it is.